packages/lib/gpcircuits/turbo.json

```diff
@@ -8,12 +8,12 @@
         "../../../node_modules/circomkit/**",
         "../../../node_modules/circomlib/**",
         "../../../node_modules/@zk-kit/circuits/**",
-        "scripts/gen-test-artifacts.ts",
+        "scripts/gen-artifacts.ts",
```

Comment:
Prod circuit/artifact directories need to show up in this file in most of the same places as test directories.  Given the env-variable config, tests can depend on prod artifacts.

Reply:
I think this is covered okay in its current state, assuming we're deciding that there's no need for turbo caching or auto-dependencies.  That means anyone building prod artifacts, or running tests with them is responsible for doing it manually in the right order.  The dependencies as-is should work for the normal case of running utests which need test artifacts.